### DISCUSSION TITLE
Using app code in background script

### DISCUSSION THREAD

**Discussion by Zummek** [quasarframework/quasar#10105] [CLI - BEX mode]
Hello,
I'm trying to use/import module from app code (/src/api.ts) in background script(/bex-src/background.js). 
Module in app code is written in Typescript. Is it possible to configure the project to use module from app code so as not to duplicate the code?

Someone described a similar problem on [stack](https://stackoverflow.com/questions/67884698/use-quasar-app-code-in-background-script-of-bex)

**Comment by swtanggara** [comment]
Hi @Zummek , I don't have any advance knowledge about webpack plumbing to configuring this. But you can manage this `use/import` requirement from `src-bex/js/background-hooks.js`.
